scio-core/src/it/scala/com/spotify/scio/testing/util/ItUtils.scala

```diff
@@ -35,11 +34,14 @@ import org.apache.beam.sdk.util.{RetryHttpRequestInitializer, Transport}
 /** Integration test utilities. */
 private[scio] object ItUtils {
 
+  // XXX: Copied from BigQueryClient
+  val PROJECT_KEY: String = "bigquery.project"
```

Comment:
Why?

Reply:
To avoid a cyclic dependencies between projects.